common/djangoapps/third_party_auth/tests/test_provider.py

```diff
@@ -114,7 +114,7 @@ def test_providers_displayed_for_login(self):
         assert no_log_in_provider.provider_id not in provider_ids
         assert normal_provider.provider_id in provider_ids
 
-    def test_tpa_hint_provider_displayed_for_login(self):
+    def test_tpa_hint_hidden_provider_displayed_for_login(self):
```

Comment:
Why did we need to break up these tests ?

Reply:
Due to some weird caching behaviour, see https://github.com/eduNEXT/edunext-platform/pull/601#discussion_r772032469

Reply:
I see. Breaking up the tests doesn't matter all that much, but we might need to justify this to upstream.
Perhaps try running the tests without this change, to see if the caching issue is still relevant ?

Reply:
Even I did not break them initially but it still has the issue.

Reply:
Sounds good.